test/mjsunit/wasm/asm-wasm.js

```diff
@@ -512,48 +512,42 @@ function TestModDoubleNegative() {
 assertEquals(28, WASM.asmCompileRun(TestModDoubleNegative.toString()));
 */
 
-function TestGlobals() {
```

Comment:
The TestGlobals function doesn't work anymore as the __init__ function is the new last function and we can't call asmCompileRun anymore for that.